RelationalAI/Rest.cs

```diff
@@ -163,6 +170,34 @@ public MetadataInfo ReadMetadataProtobuf(byte[] data)
             return MetadataInfo.Parser.ParseFrom(data);
         }
 
+        public async Task<AccessToken> RequestAccessTokenAsync(ClientCredentials creds)
+        {
+            // Form the API request body.
+            var data = new Dictionary<string, string>
+            {
+                { "client_id", creds.ClientId },
+                { "client_secret", creds.ClientSecret },
+                { "audience", creds.Audience },
+                { "grant_type", "client_credentials" }
+            };
+            var resp = await RequestHelperAsync("POST", creds.ClientCredentialsUrl, data);
+            if (!(resp is string stringResponse))
```

Comment:
In what scenario does this happen?

Reply:
I'm not the one who implemented the body of this method `RequestAccessTokenAsync`, just making it public so that the `IAccessTokenHandler` implementations can use it to fetch new tokens